google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -131,3 +146,16 @@ def get_current_span():
 def add_span_event(span, event_name, event_attributes=None):
     if span:
         span.add_event(event_name, event_attributes)
+
+
+def add_event_on_current_span(event_name, event_attributes=None, span=None):
+    if not span:
+        span = get_current_span()
+
+    add_span_event(span, event_name, event_attributes)
+
+
+def record_span_exception_and_status(span, exc):
+    if span:
+        span.set_status(Status(StatusCode.ERROR, str(exc)))
+        span.record_exception(exc)
```

Comment:
Not really, because wherever we invoke this code that's when an exception was created. It would an unnecessary pedantic step, but I can add it if you feel strongly about it.